frontend/README.md

```diff
@@ -46,22 +61,67 @@ You will now have the backend API available on `localhost:8000/docs` and the fro
-#### Running against live backend
 
-For FE development, it is probably easier to hit the live instance of circlesapi and run the FE locally. You can do this by doing:
+#### Removing Docker Containers
+
+To remove all containers and the docker network, run `docker-compose down`. Add the option `-v` to also remove persistent volumes: that is, the mongoDB data. I tend to run this before I rebuild if I have made changes to the backend code to keep everything clean on my system.
+
+To stop a docker containers that is running, run `docker-compose stop <containerName>`. This will not remove the container.
+
+## Project Structure
 ```bash
-npm install # install dependencies
-npm start
+frontend/
+├─ public/                              # files related to the initial static html
+├─ src/
+│  ├─ assets/                           # stores all static assets (images, svgs, css)
+│  ├─ components/                       # stores reusable components
+│  │  ├─ CourseButton/
+│  │  │  ├─ CourseButton.tsx            # component implementation
+│  │  │  ├─ CourseButton.test.tsx       # test file for the component
+│  │  │  ├─ index.ts                    # should contain only imports of component(s) to be exported from this folder
+│  │  │  ├─ styles.ts                   # contains stylings related to this component(s)
+│  ├─ config/                           # stores any config and misc files
+│  ├─ hooks/                            # stores custom hooks
+│  ├─ pages/
+│  │  ├─ CourseSelector/
+│  │  │  ├─ CourseSelector.tsx
+│  │  │  ├─ CourseSelector.test.tsx
+│  │  │  ├─ index.ts
+│  │  │  ├─ styles.ts
+│  │  │  ├─ CourseBanner/               # components specific to this page
+│  │  │  │  ├─ CourseBanner.tsx
+│  │  │  │  ├─ CourseBanner.test.tsx
+│  │  │  │  ├─ index.ts
+│  │  │  │  ├─ styles.ts
+│  │  │  ├─ ...
+│  │  ├─ ...
+│  ├─ reducers/                         # reducer slices files
+│  ├─ styles/                           # styled components styles that can be reused globally
+│  ├─ test/                             # helper files relating to testing
+│  ├─ types/                            # type definition files to be used globally
+│  ├─ utils/                            # utility/helper functions to be used globally
+│  ├─ App.tsx
+│  ├─ index.tsx
+│  ├─ ...
+├─ dev.dockerfile                       # dockerfile to run in dev env
+├─ production.dockerfile                # dockerfile to run in prod env
+├─ index.html
+├─ package.json
+├─ package-lock.json
+├─ tsconfig.json                        # specifies the root files and the compiler options required to compile the project
+├─ vite.config.ts                       # config file to setup vite
+├─ vitest.config.ts                     # config file to setup vitest
+├─ .eslintrc                            # eslint config
+├─ .prettierrc                          # prettier config
+├─ ...
```

Comment:
could there be a comment along the lines of `accurate as of <insert date>` for the folder structure

Reply:
@leonardo-fan fix this coz i will not